presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -114,6 +114,7 @@
     private boolean assumeCanonicalPartitionKeys;
 
     private boolean useOrcColumnNames;
+    private boolean useOrcBloomfilters = true;
```

Comment:
Name this `orcBloomfiltersEnabled` to match things like `parquetPredicatePushdownEnabled`